docs/variants/msi_z690/recovery.md

```diff
@@ -44,3 +44,66 @@ $ flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -w 7D25v13/E7D25IMS.13
 ```
 
 * First boot after the recovery process is significantly longer
+
+### SMBIOS unique data recovery
+
+#### Serial number
+
+SMBIOS specification defines two spaces for serial number: the system serial
+number and baseboard serial number. The original MSI PRO Z690-A firmware
+provides only baseboard serial number.
+
+In case you have lost your serial number in the process of flashing Dasharo or
+newer MSI firmware, there is a way to retrieve it. The board has a QR code
+imprinted between the chipset heatsink and dPGU PCIe slot:
```

Comment:
I think this part of recovering serial number is separate thing which we advertise as feature of Dasharo, which proves quality of the work. We should have specific section for serial number recovery for MSI PRO Z690-A that we can point to e.g. BIOS modders community.

Reply:
This is a recovery section, isn't it?